src/mame/mame.lst

```diff
@@ -14560,6 +14560,8 @@ pacmanbl                        // bootleg
 pacmanbla                       // bootleg
 pacmanblb                       // bootleg
 pacmanblc                       // bootleg (Calfesa)
+pacmanblci                      // bootleg (Cirsa)
+komemokos                       // hack
```

Comment:
Short names need to be sorted to keep these lists manageable.

Reply:
Ups! Fixed